[2023-06-18 16:56:22 main.py:<module>:43 INFO] servive start listening on 127.0.0.1:8090
[2023-06-18 16:56:37 main.py:<module>:43 INFO] servive start listening on 127.0.0.1:8090
[2023-06-18 16:56:49 main.py:<module>:43 INFO] servive start listening on 127.0.0.1:8090
[2023-06-18 16:59:18 main.py:<module>:43 INFO] servive start listening on 127.0.0.1:8090
[2023-06-18 16:59:25 base_handler.py:parse_json_arguments:230 INFO] request_body:b''
[2023-06-18 16:59:25 base_handler.py:log_exception:67 WARNING] [0;31mMissing argument log[0m
[2023-06-18 16:59:25 main.py:log_function:25 WARNING]  14.23ms 400   POST       127.0.0.1  /index 
[2023-06-18 16:59:25 base_handler.py:parse_json_arguments:230 INFO] request_body:b''
[2023-06-18 16:59:25 base_handler.py:log_exception:67 WARNING] [0;31mMissing argument log[0m
[2023-06-18 16:59:25 main.py:log_function:25 WARNING]   1.61ms 400   POST       127.0.0.1  /index 
[2023-06-18 17:00:15 autoreload.py:_check_file:200 INFO] /Users/van/projects/likn/event-tracker/config/config.py modified; restarting server
[2023-06-18 17:00:15 main.py:<module>:43 INFO] servive start listening on 127.0.0.1:8090
[2023-06-18 17:01:04 main.py:<module>:43 INFO] servive start listening on 127.0.0.1:8090
[2023-06-18 17:01:15 main.py:<module>:43 INFO] servive start listening on 127.0.0.1:8090
[2023-06-18 17:09:16 main.py:<module>:43 INFO] servive start listening on 127.0.0.1:8090
[2023-06-18 17:21:40 main.py:<module>:43 INFO] servive start listening on 127.0.0.1:8090
[2023-06-18 17:22:18 autoreload.py:_check_file:200 INFO] /Users/van/projects/likn/event-tracker/lib/logger.py modified; restarting server
[2023-06-18 17:22:18 main.py:<module>:43 INFO] servive start listening on 127.0.0.1:8090
[2023-06-18 17:22:20 base_handler.py:parse_json_arguments:230 INFO] request_body:b''
[2023-06-18 17:22:20 base_handler.py:log_exception:67 WARNING] [0;31mMissing argument log[0m
[2023-06-18 17:22:20 main.py:log_function:25 WARNING]  10.99ms 400   POST       127.0.0.1  /index 
[2023-06-18 17:37:53 autoreload.py:_check_file:200 INFO] /Users/van/projects/likn/event-tracker/lib/logger.py modified; restarting server
[2023-06-18 17:37:54 main.py:<module>:43 INFO] servive start listening on 127.0.0.1:8090
[32m[2023-06-18 17:38:05 main.py:<module>:43 INFO](B[m servive start listening on 127.0.0.1:8090
[32m[2023-06-18 17:38:22 base_handler.py:parse_json_arguments:230 INFO](B[m request_body:b''
[33m[2023-06-18 17:38:22 base_handler.py:log_exception:67 WARNING](B[m [0;31mMissing argument log[0m
[33m[2023-06-18 17:38:22 main.py:log_function:25 WARNING](B[m   6.98ms 400   POST       127.0.0.1  /index 
[32m[2023-06-18 17:38:23 base_handler.py:parse_json_arguments:230 INFO](B[m request_body:b''
[33m[2023-06-18 17:38:23 base_handler.py:log_exception:67 WARNING](B[m [0;31mMissing argument log[0m
[33m[2023-06-18 17:38:23 main.py:log_function:25 WARNING](B[m   0.63ms 400   POST       127.0.0.1  /index 
[2023-06-18 17:40:45 main.py:<module>:43 INFO] servive start listening on 127.0.0.1:8090
[2023-06-18 17:40:55 base_handler.py:parse_json_arguments:230 INFO] request_body:b''
[2023-06-18 17:40:55 base_handler.py:log_exception:67 WARNING] [0;31mMissing argument log[0m
[2023-06-18 17:40:55 main.py:log_function:25 WARNING]  14.15ms 400   POST       127.0.0.1  /index 
[2023-06-18 17:40:56 base_handler.py:parse_json_arguments:230 INFO] request_body:b''
[2023-06-18 17:40:56 base_handler.py:log_exception:67 WARNING] [0;31mMissing argument log[0m
[2023-06-18 17:40:56 main.py:log_function:25 WARNING]   2.25ms 400   POST       127.0.0.1  /index 
[2023-06-18 17:50:57 main.py:<module>:43 INFO] servive start listening on 127.0.0.1:8090
[2023-06-18 17:51:12 main.py:<module>:43 INFO] servive start listening on 127.0.0.1:8090
[2023-06-18 17:59:30 base_handler.py:parse_json_arguments:230 INFO] request_body:b''
[2023-06-18 17:59:30 base_handler.py:log_exception:67 WARNING] [0;31mMissing argument log[0m
[2023-06-18 17:59:30 main.py:log_function:25 WARNING]   9.10ms 400   POST       127.0.0.1  /index 
[2023-06-18 17:59:31 base_handler.py:parse_json_arguments:230 INFO] request_body:b''
[2023-06-18 17:59:31 base_handler.py:log_exception:67 WARNING] [0;31mMissing argument log[0m
[2023-06-18 17:59:31 main.py:log_function:25 WARNING]   1.39ms 400   POST       127.0.0.1  /index 
[2023-06-18 18:01:10 main.py:<module>:43 INFO] servive start listening on 127.0.0.1:8090
[2023-06-18 18:01:14 base_handler.py:parse_json_arguments:230 INFO] request_body:b''
[2023-06-18 18:01:14 base_handler.py:log_exception:67 WARNING] [0;31mMissing argument log[0m
[2023-06-18 18:01:14 main.py:log_function:25 WARNING]   6.80ms 400   POST       127.0.0.1  /index 
[2023-06-18 18:01:32 main.py:<module>:43 INFO] servive start listening on 127.0.0.1:8090
[2023-06-18 18:01:36 base_handler.py:parse_json_arguments:230 INFO] request_body:b''
[2023-06-18 21:13:49 main.py:<module>:43 INFO] servive start listening on 127.0.0.1:8090
[2023-06-18 21:14:36 base_handler.py:parse_json_arguments:231 INFO] request_body:b''
[2023-06-18 21:15:18 base_handler.py:log_exception:68 WARNING] [0;31mMissing argument log[0m
[2023-06-18 21:15:18 main.py:log_function:25 WARNING] 41544.98ms 400   POST       127.0.0.1  /index 
[2023-06-18 21:25:47 autoreload.py:_check_file:200 INFO] /Users/van/projects/likn/event-tracker/handlers/base_handler.py modified; restarting server
[2023-06-18 21:25:48 main.py:<module>:43 INFO] servive start listening on 127.0.0.1:8090
[2023-06-18 21:26:05 main.py:<module>:43 INFO] servive start listening on 127.0.0.1:8090
[2023-06-18 21:26:10 base_handler.py:parse_json_arguments:230 INFO] request_body:b''
[2023-06-18 21:26:10 base_handler.py:log_exception:67 WARNING] 400   POST       127.0.0.1  /index : Missing argument log
[2023-06-18 21:26:10 main.py:log_function:25 WARNING]   9.03ms 400   POST       127.0.0.1  /index 
[2023-06-18 21:26:11 base_handler.py:parse_json_arguments:230 INFO] request_body:b''
[2023-06-18 21:26:11 base_handler.py:log_exception:67 WARNING] 400   POST       127.0.0.1  /index : Missing argument log
[2023-06-18 21:26:11 main.py:log_function:25 WARNING]   2.62ms 400   POST       127.0.0.1  /index 
[2023-06-18 21:26:54 base_handler.py:parse_json_arguments:230 INFO] request_body:b''
[2023-06-18 21:26:54 base_handler.py:log_exception:67 WARNING] 400   POST       127.0.0.1  /index : Missing argument log
[2023-06-18 21:26:54 main.py:log_function:25 WARNING]   0.71ms 400   POST       127.0.0.1  /index 
[2023-06-18 21:28:42 main.py:<module>:43 INFO] servive start listening on 127.0.0.1:8090
[2023-06-18 21:28:45 base_handler.py:parse_json_arguments:230 INFO] request_body:b''
[2023-06-18 21:28:45 base_handler.py:log_exception:68 WARNING] [0;31mMissing argument log[0m
[2023-06-18 21:28:45 main.py:log_function:25 WARNING]   6.40ms 400   POST       127.0.0.1  /index 
[2023-06-18 21:30:06 autoreload.py:_check_file:200 INFO] /Users/van/projects/likn/event-tracker/handlers/base_handler.py modified; restarting server
[2023-06-18 21:30:06 main.py:<module>:43 INFO] servive start listening on 127.0.0.1:8090
[2023-06-18 21:35:11 autoreload.py:_check_file:200 INFO] /Users/van/projects/likn/event-tracker/handlers/index.py modified; restarting server
[2023-06-18 21:35:11 main.py:<module>:43 INFO] servive start listening on 127.0.0.1:8090
[2023-06-18 21:35:27 autoreload.py:_check_file:200 INFO] /Users/van/projects/likn/event-tracker/handlers/index.py modified; restarting server
[2023-06-18 21:35:27 main.py:<module>:43 INFO] servive start listening on 127.0.0.1:8090
[2023-06-18 21:37:18 main.py:<module>:43 INFO] servive start listening on 127.0.0.1:8090
[2023-06-18 21:37:25 base_handler.py:parse_json_arguments:230 INFO] request_body:b''
[2023-06-18 21:37:25 base_handler.py:log_exception:67 WARNING] 400   POST       127.0.0.1  /index : Missing argument log
[2023-06-18 21:37:25 main.py:log_function:25 WARNING]  16.46ms 400   POST       127.0.0.1  /index 
[2023-06-18 21:55:09 autoreload.py:_check_file:200 INFO] /Users/van/projects/likn/event-tracker/handlers/base_handler.py modified; restarting server
[2023-06-18 21:55:09 main.py:<module>:43 INFO] servive start listening on 127.0.0.1:8090
[2023-06-18 22:05:37 main.py:<module>:43 INFO] servive start listening on 127.0.0.1:8090
[2023-06-18 22:05:42 base_handler.py:parse_json_arguments:232 INFO] request_body:b''
[2023-06-18 22:05:42 base_handler.py:log_exception:69 WARNING] 400   POST       127.0.0.1  /index : Missing argument 'log'
[2023-06-18 22:05:42 main.py:log_function:25 WARNING]  11.72ms 400   POST       127.0.0.1  /index 
[2023-06-18 22:07:01 base_handler.py:parse_json_arguments:232 INFO] request_body:b''
[2023-06-18 22:07:01 base_handler.py:log_exception:69 WARNING] 400   POST       127.0.0.1  /index : Missing argument 'log'
[2023-06-18 22:07:01 main.py:log_function:25 WARNING]   2.66ms 400   POST       127.0.0.1  /index 
[2023-06-18 22:07:25 base_handler.py:parse_json_arguments:232 INFO] request_body:b'{\n    "log":""\n}'
[2023-06-18 22:07:25 base_handler.py:parse_json_arguments:232 INFO] request_body:b'{\n    "log":""\n}'
[2023-06-18 22:07:25 base_handler.py:log_exception:69 WARNING] 400   POST       127.0.0.1  /index : Missing argument 'i'
[2023-06-18 22:07:25 main.py:log_function:25 WARNING]   1.47ms 400   POST       127.0.0.1  /index 
[2023-06-18 22:07:35 base_handler.py:parse_json_arguments:232 INFO] request_body:b'{\n    "log":"",\n    "i":""\n}'
[2023-06-18 22:07:35 base_handler.py:parse_json_arguments:232 INFO] request_body:b'{\n    "log":"",\n    "i":""\n}'
[2023-06-18 22:07:35 base_handler.py:parse_json_arguments:232 INFO] request_body:b'{\n    "log":"",\n    "i":""\n}'
[2023-06-18 22:07:35 base_handler.py:log_exception:69 WARNING] 400   POST       127.0.0.1  /index : Missing argument 'user_id'
[2023-06-18 22:07:35 main.py:log_function:25 WARNING]   2.19ms 400   POST       127.0.0.1  /index 
[2023-06-18 22:09:34 main.py:<module>:43 INFO] servive start listening on 127.0.0.1:8090
[2023-06-18 22:09:41 base_handler.py:parse_json_arguments:232 INFO] request_body:b'{\n    "log":"",\n    "i":""\n}'
[2023-06-18 22:10:06 base_handler.py:parse_json_arguments:232 INFO] request_body:b'{\n    "log":"",\n    "i":""\n}'
[2023-06-18 22:10:33 base_handler.py:parse_json_arguments:232 INFO] request_body:b'{\n    "log":"",\n    "i":""\n}'
[2023-06-18 22:13:08 base_handler.py:log_exception:69 WARNING] 400   POST       127.0.0.1  /index : Missing argument 'user_id'
[2023-06-18 22:16:40 main.py:log_function:25 WARNING] 388966.63ms 400   POST       127.0.0.1  /index 
[2023-06-18 22:19:30 main.py:<module>:43 INFO] servive start listening on 127.0.0.1:8090
[2023-06-18 22:19:34 base_handler.py:parse_json_arguments:232 INFO] request_body:b'{\n    "log":"",\n    "i":""\n}'
[2023-06-18 22:19:40 base_handler.py:parse_json_arguments:232 INFO] request_body:b'{\n    "log":"",\n    "i":""\n}'
[2023-06-18 22:19:41 base_handler.py:parse_json_arguments:232 INFO] request_body:b'{\n    "log":"",\n    "i":""\n}'
[2023-06-18 22:19:46 base_handler.py:log_exception:69 WARNING] 400   POST       127.0.0.1  /index : Missing argument 'user_id'
[2023-06-18 22:24:43 main.py:<module>:44 INFO] servive start listening on 127.0.0.1:8090
[2023-06-18 22:24:47 base_handler.py:parse_json_arguments:232 INFO] request_body:b'{\n    "log":"",\n    "i":""\n}'
[2023-06-18 22:24:47 base_handler.py:parse_json_arguments:232 INFO] request_body:b'{\n    "log":"",\n    "i":""\n}'
[2023-06-18 22:24:47 base_handler.py:parse_json_arguments:232 INFO] request_body:b'{\n    "log":"",\n    "i":""\n}'
[2023-06-18 22:24:47 base_handler.py:log_exception:69 WARNING] 400   POST       127.0.0.1  /index : Missing argument 'user_id'
[2023-06-18 22:24:47 main.py:log_function:25 WARNING]   2.16ms 400   POST       127.0.0.1  /index 
[2023-06-18 22:24:50 base_handler.py:parse_json_arguments:232 INFO] request_body:b'{\n    "log":"",\n    "i":""\n}'
[2023-06-18 22:24:50 base_handler.py:parse_json_arguments:232 INFO] request_body:b'{\n    "log":"",\n    "i":""\n}'
[2023-06-18 22:24:50 base_handler.py:parse_json_arguments:232 INFO] request_body:b'{\n    "log":"",\n    "i":""\n}'
[2023-06-18 22:24:50 base_handler.py:log_exception:69 WARNING] 400   POST       127.0.0.1  /index : Missing argument 'user_id'
[2023-06-18 22:24:50 main.py:log_function:25 WARNING]   2.87ms 400   POST       127.0.0.1  /index 
[2023-06-18 22:24:50 base_handler.py:parse_json_arguments:232 INFO] request_body:b'{\n    "log":"",\n    "i":""\n}'
[2023-06-18 22:24:50 base_handler.py:parse_json_arguments:232 INFO] request_body:b'{\n    "log":"",\n    "i":""\n}'
[2023-06-18 22:24:50 base_handler.py:parse_json_arguments:232 INFO] request_body:b'{\n    "log":"",\n    "i":""\n}'
[2023-06-18 22:24:50 base_handler.py:log_exception:69 WARNING] 400   POST       127.0.0.1  /index : Missing argument 'user_id'
[2023-06-18 22:24:50 main.py:log_function:25 WARNING]   1.28ms 400   POST       127.0.0.1  /index 
[2023-06-18 22:24:52 base_handler.py:parse_json_arguments:232 INFO] request_body:b'{\n    "log":"",\n    "i":""\n}'
[2023-06-18 22:24:52 base_handler.py:parse_json_arguments:232 INFO] request_body:b'{\n    "log":"",\n    "i":""\n}'
[2023-06-18 22:24:52 base_handler.py:parse_json_arguments:232 INFO] request_body:b'{\n    "log":"",\n    "i":""\n}'
[2023-06-18 22:24:52 base_handler.py:log_exception:69 WARNING] 400   POST       127.0.0.1  /index : Missing argument 'user_id'
[2023-06-18 22:24:52 main.py:log_function:25 WARNING]   2.91ms 400   POST       127.0.0.1  /index 
[2023-06-18 22:24:54 base_handler.py:parse_json_arguments:232 INFO] request_body:b'{\n    "log":"",\n    "i":""\n}'
[2023-06-18 22:24:54 base_handler.py:parse_json_arguments:232 INFO] request_body:b'{\n    "log":"",\n    "i":""\n}'
[2023-06-18 22:24:54 base_handler.py:parse_json_arguments:232 INFO] request_body:b'{\n    "log":"",\n    "i":""\n}'
[2023-06-18 22:24:54 base_handler.py:log_exception:69 WARNING] 400   POST       127.0.0.1  /index : Missing argument 'user_id'
[2023-06-18 22:24:54 main.py:log_function:25 WARNING]   4.80ms 400   POST       127.0.0.1  /index 
[2023-06-18 22:24:57 base_handler.py:parse_json_arguments:232 INFO] request_body:b'{\n    "log":"",\n    "i":""\n}'
[2023-06-18 22:24:57 base_handler.py:parse_json_arguments:232 INFO] request_body:b'{\n    "log":"",\n    "i":""\n}'
[2023-06-18 22:24:57 base_handler.py:parse_json_arguments:232 INFO] request_body:b'{\n    "log":"",\n    "i":""\n}'
[2023-06-18 22:24:57 base_handler.py:log_exception:69 WARNING] 400   POST       127.0.0.1  /index : Missing argument 'user_id'
[2023-06-18 22:24:57 main.py:log_function:25 WARNING]   1.93ms 400   POST       127.0.0.1  /index 
[2023-06-18 22:27:17 main.py:<module>:44 INFO] servive start listening on 127.0.0.1:8090
[2023-06-18 22:27:24 base_handler.py:parse_json_arguments:232 INFO] request_body:b'{\n    "log":"",\n    "i":""\n}'
[2023-06-18 22:27:26 base_handler.py:parse_json_arguments:232 INFO] request_body:b'{\n    "log":"",\n    "i":""\n}'
[2023-06-18 22:27:27 base_handler.py:parse_json_arguments:232 INFO] request_body:b'{\n    "log":"",\n    "i":""\n}'
[2023-06-18 22:27:29 base_handler.py:log_exception:69 WARNING] 400   POST       127.0.0.1  /index : Missing argument 'user_id'
[2023-06-18 22:30:51 main.py:<module>:44 INFO] servive start listening on 127.0.0.1:8090
[2023-06-18 22:30:54 base_handler.py:parse_json_arguments:232 INFO] request_body:b'{\n    "log":"",\n    "i":""\n}'
[2023-06-18 22:30:54 base_handler.py:parse_json_arguments:232 INFO] request_body:b'{\n    "log":"",\n    "i":""\n}'
[2023-06-18 22:30:54 base_handler.py:parse_json_arguments:232 INFO] request_body:b'{\n    "log":"",\n    "i":""\n}'
[2023-06-18 22:30:54 base_handler.py:log_exception:69 WARNING] 400   POST       127.0.0.1  /index : Missing argument 'user_id'
[2023-06-18 22:30:54 web.py:send_error:1296 ERROR] Uncaught exception in write_error
    Traceback (most recent call last):
      File "/Users/van/projects/likn/event-tracker/venv/lib/python3.9/site-packages/tornado/web.py", line 1786, in _execute
        result = await result
      File "/Users/van/projects/likn/event-tracker/venv/lib/python3.9/site-packages/tornado/gen.py", line 234, in wrapper
        yielded = ctx_run(next, result)
      File "/Users/van/projects/likn/event-tracker/handlers/index.py", line 22, in post
        args = self.parse_json_arguments(
      File "/Users/van/projects/likn/event-tracker/handlers/base_handler.py", line 252, in parse_json_arguments
        raise MissingArgumentError(f"'{key}'")
    tornado.web.MissingArgumentError: HTTP 400: Bad Request (Missing argument 'user_id')
    
    During handling of the above exception, another exception occurred:
    
    Traceback (most recent call last):
      File "/Users/van/projects/likn/event-tracker/venv/lib/python3.9/site-packages/tornado/web.py", line 1294, in send_error
        self.write_error(status_code, **kwargs)
      File "/Users/van/projects/likn/event-tracker/handlers/base_handler.py", line 315, in write_error
        json.dumps(
      File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/json/__init__.py", line 231, in dumps
        return _default_encoder.encode(obj)
      File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/json/encoder.py", line 199, in encode
        chunks = self.iterencode(o, _one_shot=True)
      File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/json/encoder.py", line 257, in iterencode
        return _iterencode(o, 0)
      File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/json/encoder.py", line 179, in default
        raise TypeError(f'Object of type {o.__class__.__name__} '
    TypeError: Object of type set is not JSON serializable
[2023-06-18 22:30:54 main.py:log_function:25 WARNING]   3.81ms 400   POST       127.0.0.1  /index 
[2023-06-18 22:30:56 base_handler.py:parse_json_arguments:232 INFO] request_body:b'{\n    "log":"",\n    "i":""\n}'
[2023-06-18 22:30:56 base_handler.py:parse_json_arguments:232 INFO] request_body:b'{\n    "log":"",\n    "i":""\n}'
[2023-06-18 22:30:56 base_handler.py:parse_json_arguments:232 INFO] request_body:b'{\n    "log":"",\n    "i":""\n}'
[2023-06-18 22:30:56 base_handler.py:log_exception:69 WARNING] 400   POST       127.0.0.1  /index : Missing argument 'user_id'
[2023-06-18 22:30:56 web.py:send_error:1296 ERROR] Uncaught exception in write_error
    Traceback (most recent call last):
      File "/Users/van/projects/likn/event-tracker/venv/lib/python3.9/site-packages/tornado/web.py", line 1786, in _execute
        result = await result
      File "/Users/van/projects/likn/event-tracker/venv/lib/python3.9/site-packages/tornado/gen.py", line 234, in wrapper
        yielded = ctx_run(next, result)
      File "/Users/van/projects/likn/event-tracker/handlers/index.py", line 22, in post
        args = self.parse_json_arguments(
      File "/Users/van/projects/likn/event-tracker/handlers/base_handler.py", line 252, in parse_json_arguments
        raise MissingArgumentError(f"'{key}'")
    tornado.web.MissingArgumentError: HTTP 400: Bad Request (Missing argument 'user_id')
    
    During handling of the above exception, another exception occurred:
    
    Traceback (most recent call last):
      File "/Users/van/projects/likn/event-tracker/venv/lib/python3.9/site-packages/tornado/web.py", line 1294, in send_error
        self.write_error(status_code, **kwargs)
      File "/Users/van/projects/likn/event-tracker/handlers/base_handler.py", line 315, in write_error
        json.dumps(
      File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/json/__init__.py", line 231, in dumps
        return _default_encoder.encode(obj)
      File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/json/encoder.py", line 199, in encode
        chunks = self.iterencode(o, _one_shot=True)
      File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/json/encoder.py", line 257, in iterencode
        return _iterencode(o, 0)
      File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/json/encoder.py", line 179, in default
        raise TypeError(f'Object of type {o.__class__.__name__} '
    TypeError: Object of type set is not JSON serializable
[2023-06-18 22:30:56 main.py:log_function:25 WARNING]   0.90ms 400   POST       127.0.0.1  /index 
[2023-06-18 22:30:56 base_handler.py:parse_json_arguments:232 INFO] request_body:b'{\n    "log":"",\n    "i":""\n}'
[2023-06-18 22:30:56 base_handler.py:parse_json_arguments:232 INFO] request_body:b'{\n    "log":"",\n    "i":""\n}'
[2023-06-18 22:30:56 base_handler.py:parse_json_arguments:232 INFO] request_body:b'{\n    "log":"",\n    "i":""\n}'
[2023-06-18 22:30:56 base_handler.py:log_exception:69 WARNING] 400   POST       127.0.0.1  /index : Missing argument 'user_id'
[2023-06-18 22:30:56 web.py:send_error:1296 ERROR] Uncaught exception in write_error
    Traceback (most recent call last):
      File "/Users/van/projects/likn/event-tracker/venv/lib/python3.9/site-packages/tornado/web.py", line 1786, in _execute
        result = await result
      File "/Users/van/projects/likn/event-tracker/venv/lib/python3.9/site-packages/tornado/gen.py", line 234, in wrapper
        yielded = ctx_run(next, result)
      File "/Users/van/projects/likn/event-tracker/handlers/index.py", line 22, in post
        args = self.parse_json_arguments(
      File "/Users/van/projects/likn/event-tracker/handlers/base_handler.py", line 252, in parse_json_arguments
        raise MissingArgumentError(f"'{key}'")
    tornado.web.MissingArgumentError: HTTP 400: Bad Request (Missing argument 'user_id')
    
    During handling of the above exception, another exception occurred:
    
    Traceback (most recent call last):
      File "/Users/van/projects/likn/event-tracker/venv/lib/python3.9/site-packages/tornado/web.py", line 1294, in send_error
        self.write_error(status_code, **kwargs)
      File "/Users/van/projects/likn/event-tracker/handlers/base_handler.py", line 315, in write_error
        json.dumps(
      File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/json/__init__.py", line 231, in dumps
        return _default_encoder.encode(obj)
      File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/json/encoder.py", line 199, in encode
        chunks = self.iterencode(o, _one_shot=True)
      File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/json/encoder.py", line 257, in iterencode
        return _iterencode(o, 0)
      File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/json/encoder.py", line 179, in default
        raise TypeError(f'Object of type {o.__class__.__name__} '
    TypeError: Object of type set is not JSON serializable
[2023-06-18 22:30:56 main.py:log_function:25 WARNING]   2.13ms 400   POST       127.0.0.1  /index 
[2023-06-18 22:32:52 main.py:<module>:44 INFO] servive start listening on 127.0.0.1:8090
[2023-06-18 22:32:55 base_handler.py:parse_json_arguments:232 INFO] request_body:b'{\n    "log":"",\n    "i":""\n}'
[2023-06-18 22:32:55 base_handler.py:parse_json_arguments:232 INFO] request_body:b'{\n    "log":"",\n    "i":""\n}'
[2023-06-18 22:32:55 base_handler.py:parse_json_arguments:232 INFO] request_body:b'{\n    "log":"",\n    "i":""\n}'
[2023-06-18 22:32:55 base_handler.py:log_exception:69 WARNING] 400   POST       127.0.0.1  /index : Missing argument 'user_id'
[2023-06-18 22:32:55 web.py:send_error:1296 ERROR] Uncaught exception in write_error
    Traceback (most recent call last):
      File "/Users/van/projects/likn/event-tracker/venv/lib/python3.9/site-packages/tornado/web.py", line 1786, in _execute
        result = await result
      File "/Users/van/projects/likn/event-tracker/venv/lib/python3.9/site-packages/tornado/gen.py", line 234, in wrapper
        yielded = ctx_run(next, result)
      File "/Users/van/projects/likn/event-tracker/handlers/index.py", line 22, in post
        args = self.parse_json_arguments(
      File "/Users/van/projects/likn/event-tracker/handlers/base_handler.py", line 252, in parse_json_arguments
        raise MissingArgumentError(f"'{key}'")
    tornado.web.MissingArgumentError: HTTP 400: Bad Request (Missing argument 'user_id')
    
    During handling of the above exception, another exception occurred:
    
    Traceback (most recent call last):
      File "/Users/van/projects/likn/event-tracker/venv/lib/python3.9/site-packages/tornado/web.py", line 1294, in send_error
        self.write_error(status_code, **kwargs)
      File "/Users/van/projects/likn/event-tracker/handlers/base_handler.py", line 315, in write_error
        json.dumps(
      File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/json/__init__.py", line 231, in dumps
        return _default_encoder.encode(obj)
      File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/json/encoder.py", line 199, in encode
        chunks = self.iterencode(o, _one_shot=True)
      File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/json/encoder.py", line 257, in iterencode
        return _iterencode(o, 0)
      File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/json/encoder.py", line 179, in default
        raise TypeError(f'Object of type {o.__class__.__name__} '
    TypeError: Object of type type is not JSON serializable
[2023-06-18 22:32:55 main.py:log_function:25 WARNING]   6.88ms 400   POST       127.0.0.1  /index 
[2023-06-18 22:32:56 base_handler.py:parse_json_arguments:232 INFO] request_body:b'{\n    "log":"",\n    "i":""\n}'
[2023-06-18 22:32:56 base_handler.py:parse_json_arguments:232 INFO] request_body:b'{\n    "log":"",\n    "i":""\n}'
[2023-06-18 22:32:56 base_handler.py:parse_json_arguments:232 INFO] request_body:b'{\n    "log":"",\n    "i":""\n}'
[2023-06-18 22:32:56 base_handler.py:log_exception:69 WARNING] 400   POST       127.0.0.1  /index : Missing argument 'user_id'
[2023-06-18 22:32:56 web.py:send_error:1296 ERROR] Uncaught exception in write_error
    Traceback (most recent call last):
      File "/Users/van/projects/likn/event-tracker/venv/lib/python3.9/site-packages/tornado/web.py", line 1786, in _execute
        result = await result
      File "/Users/van/projects/likn/event-tracker/venv/lib/python3.9/site-packages/tornado/gen.py", line 234, in wrapper
        yielded = ctx_run(next, result)
      File "/Users/van/projects/likn/event-tracker/handlers/index.py", line 22, in post
        args = self.parse_json_arguments(
      File "/Users/van/projects/likn/event-tracker/handlers/base_handler.py", line 252, in parse_json_arguments
        raise MissingArgumentError(f"'{key}'")
    tornado.web.MissingArgumentError: HTTP 400: Bad Request (Missing argument 'user_id')
    
    During handling of the above exception, another exception occurred:
    
    Traceback (most recent call last):
      File "/Users/van/projects/likn/event-tracker/venv/lib/python3.9/site-packages/tornado/web.py", line 1294, in send_error
        self.write_error(status_code, **kwargs)
      File "/Users/van/projects/likn/event-tracker/handlers/base_handler.py", line 315, in write_error
        json.dumps(
      File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/json/__init__.py", line 231, in dumps
        return _default_encoder.encode(obj)
      File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/json/encoder.py", line 199, in encode
        chunks = self.iterencode(o, _one_shot=True)
      File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/json/encoder.py", line 257, in iterencode
        return _iterencode(o, 0)
      File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/json/encoder.py", line 179, in default
        raise TypeError(f'Object of type {o.__class__.__name__} '
    TypeError: Object of type type is not JSON serializable
[2023-06-18 22:32:56 main.py:log_function:25 WARNING]   0.71ms 400   POST       127.0.0.1  /index 
[2023-06-18 22:33:18 main.py:<module>:44 INFO] servive start listening on 127.0.0.1:8090
[2023-06-18 22:33:22 base_handler.py:parse_json_arguments:232 INFO] request_body:b'{\n    "log":"",\n    "i":""\n}'
[2023-06-18 22:33:22 base_handler.py:parse_json_arguments:232 INFO] request_body:b'{\n    "log":"",\n    "i":""\n}'
[2023-06-18 22:33:22 base_handler.py:parse_json_arguments:232 INFO] request_body:b'{\n    "log":"",\n    "i":""\n}'
[2023-06-18 22:33:22 base_handler.py:log_exception:69 WARNING] 400   POST       127.0.0.1  /index : Missing argument 'user_id'
[2023-06-18 22:33:22 web.py:send_error:1296 ERROR] Uncaught exception in write_error
    Traceback (most recent call last):
      File "/Users/van/projects/likn/event-tracker/venv/lib/python3.9/site-packages/tornado/web.py", line 1786, in _execute
        result = await result
      File "/Users/van/projects/likn/event-tracker/venv/lib/python3.9/site-packages/tornado/gen.py", line 234, in wrapper
        yielded = ctx_run(next, result)
      File "/Users/van/projects/likn/event-tracker/handlers/index.py", line 22, in post
        args = self.parse_json_arguments(
      File "/Users/van/projects/likn/event-tracker/handlers/base_handler.py", line 252, in parse_json_arguments
        raise MissingArgumentError(f"'{key}'")
    tornado.web.MissingArgumentError: HTTP 400: Bad Request (Missing argument 'user_id')
    
    During handling of the above exception, another exception occurred:
    
    Traceback (most recent call last):
      File "/Users/van/projects/likn/event-tracker/venv/lib/python3.9/site-packages/tornado/web.py", line 1294, in send_error
        self.write_error(status_code, **kwargs)
      File "/Users/van/projects/likn/event-tracker/handlers/base_handler.py", line 316, in write_error
        json.dumps(
      File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/json/__init__.py", line 231, in dumps
        return _default_encoder.encode(obj)
      File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/json/encoder.py", line 199, in encode
        chunks = self.iterencode(o, _one_shot=True)
      File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/json/encoder.py", line 257, in iterencode
        return _iterencode(o, 0)
      File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/json/encoder.py", line 179, in default
        raise TypeError(f'Object of type {o.__class__.__name__} '
    TypeError: Object of type type is not JSON serializable
[2023-06-18 22:33:22 main.py:log_function:25 WARNING]   4.35ms 400   POST       127.0.0.1  /index 
[2023-06-18 22:35:02 main.py:<module>:44 INFO] servive start listening on 127.0.0.1:8090
[2023-06-18 22:35:05 base_handler.py:parse_json_arguments:232 INFO] request_body:b'{\n    "log":"",\n    "i":""\n}'
[2023-06-18 22:35:05 base_handler.py:parse_json_arguments:232 INFO] request_body:b'{\n    "log":"",\n    "i":""\n}'
[2023-06-18 22:35:05 base_handler.py:parse_json_arguments:232 INFO] request_body:b'{\n    "log":"",\n    "i":""\n}'
[2023-06-18 22:35:05 base_handler.py:log_exception:69 WARNING] 400   POST       127.0.0.1  /index : Missing argument 'user_id'
[2023-06-18 22:35:05 main.py:log_function:25 WARNING]   1.59ms 400   POST       127.0.0.1  /index 
[2023-06-18 22:35:07 base_handler.py:parse_json_arguments:232 INFO] request_body:b'{\n    "log":"",\n    "i":""\n}'
[2023-06-18 22:35:07 base_handler.py:parse_json_arguments:232 INFO] request_body:b'{\n    "log":"",\n    "i":""\n}'
[2023-06-18 22:35:07 base_handler.py:parse_json_arguments:232 INFO] request_body:b'{\n    "log":"",\n    "i":""\n}'
[2023-06-18 22:35:07 base_handler.py:log_exception:69 WARNING] 400   POST       127.0.0.1  /index : Missing argument 'user_id'
[2023-06-18 22:35:07 main.py:log_function:25 WARNING]   1.83ms 400   POST       127.0.0.1  /index 
[2023-06-18 22:39:03 main.py:<module>:44 INFO] servive start listening on 127.0.0.1:8090
[2023-06-18 22:39:05 base_handler.py:parse_json_arguments:232 INFO] request_body:{'log': '', 'i': ''}
[2023-06-18 22:39:05 base_handler.py:parse_json_arguments:232 INFO] request_body:{'log': '', 'i': ''}
[2023-06-18 22:39:05 base_handler.py:parse_json_arguments:232 INFO] request_body:{'log': '', 'i': ''}
[2023-06-18 22:39:05 base_handler.py:log_exception:69 WARNING] 400   POST       127.0.0.1  /index : Missing argument 'user_id'
[2023-06-18 22:39:05 main.py:log_function:25 WARNING]   3.44ms 400   POST       127.0.0.1  /index 
[2023-06-18 22:41:59 main.py:<module>:44 INFO] servive start listening on 127.0.0.1:8090
[2023-06-18 22:42:02 base_handler.py:log_exception:72 ERROR] Uncaught exception   POST       127.0.0.1  /index 
    HTTPServerRequest(protocol='http', host='127.0.0.1:8090', method='POST', uri='/index?log', version='HTTP/1.1', remote_ip='127.0.0.1')
    Traceback (most recent call last):
      File "/Users/van/projects/likn/event-tracker/venv/lib/python3.9/site-packages/tornado/web.py", line 1786, in _execute
        result = await result
      File "/Users/van/projects/likn/event-tracker/venv/lib/python3.9/site-packages/tornado/gen.py", line 234, in wrapper
        yielded = ctx_run(next, result)
      File "/Users/van/projects/likn/event-tracker/handlers/index.py", line 23, in post
        app_log.info(json.loads(self.request.body()))
    TypeError: 'bytes' object is not callable
[2023-06-18 22:42:02 main.py:log_function:25 ERROR]   2.65ms 500   POST       127.0.0.1  /index 
[2023-06-18 22:42:04 base_handler.py:log_exception:72 ERROR] Uncaught exception   POST       127.0.0.1  /index 
    HTTPServerRequest(protocol='http', host='127.0.0.1:8090', method='POST', uri='/index?log', version='HTTP/1.1', remote_ip='127.0.0.1')
    Traceback (most recent call last):
      File "/Users/van/projects/likn/event-tracker/venv/lib/python3.9/site-packages/tornado/web.py", line 1786, in _execute
        result = await result
      File "/Users/van/projects/likn/event-tracker/venv/lib/python3.9/site-packages/tornado/gen.py", line 234, in wrapper
        yielded = ctx_run(next, result)
      File "/Users/van/projects/likn/event-tracker/handlers/index.py", line 23, in post
        app_log.info(json.loads(self.request.body()))
    TypeError: 'bytes' object is not callable
[2023-06-18 22:42:04 main.py:log_function:25 ERROR]   1.81ms 500   POST       127.0.0.1  /index 
[2023-06-18 22:42:24 main.py:<module>:44 INFO] servive start listening on 127.0.0.1:8090
[2023-06-18 22:42:27 base_handler.py:log_exception:72 ERROR] Uncaught exception   POST       127.0.0.1  /index 
    HTTPServerRequest(protocol='http', host='127.0.0.1:8090', method='POST', uri='/index?log', version='HTTP/1.1', remote_ip='127.0.0.1')
    Traceback (most recent call last):
      File "/Users/van/projects/likn/event-tracker/venv/lib/python3.9/site-packages/tornado/web.py", line 1786, in _execute
        result = await result
      File "/Users/van/projects/likn/event-tracker/venv/lib/python3.9/site-packages/tornado/gen.py", line 234, in wrapper
        yielded = ctx_run(next, result)
      File "/Users/van/projects/likn/event-tracker/handlers/index.py", line 23, in post
        app_log.info(json.loads(self.request.body().decode()))
    TypeError: 'bytes' object is not callable
[2023-06-18 22:42:27 main.py:log_function:25 ERROR]  10.11ms 500   POST       127.0.0.1  /index 
[2023-06-18 22:42:28 base_handler.py:log_exception:72 ERROR] Uncaught exception   POST       127.0.0.1  /index 
    HTTPServerRequest(protocol='http', host='127.0.0.1:8090', method='POST', uri='/index?log', version='HTTP/1.1', remote_ip='127.0.0.1')
    Traceback (most recent call last):
      File "/Users/van/projects/likn/event-tracker/venv/lib/python3.9/site-packages/tornado/web.py", line 1786, in _execute
        result = await result
      File "/Users/van/projects/likn/event-tracker/venv/lib/python3.9/site-packages/tornado/gen.py", line 234, in wrapper
        yielded = ctx_run(next, result)
      File "/Users/van/projects/likn/event-tracker/handlers/index.py", line 23, in post
        app_log.info(json.loads(self.request.body().decode()))
    TypeError: 'bytes' object is not callable
[2023-06-18 22:42:28 main.py:log_function:25 ERROR]   0.53ms 500   POST       127.0.0.1  /index 
[2023-06-18 22:42:45 main.py:<module>:44 INFO] servive start listening on 127.0.0.1:8090
[2023-06-18 22:42:47 index.py:post:23 INFO] {'log': '', 'i': ''}
[2023-06-18 22:42:47 base_handler.py:parse_json_arguments:232 INFO] request_body:{'log': '', 'i': ''}
[2023-06-18 22:42:47 base_handler.py:parse_json_arguments:232 INFO] request_body:{'log': '', 'i': ''}
[2023-06-18 22:42:47 base_handler.py:parse_json_arguments:232 INFO] request_body:{'log': '', 'i': ''}
[2023-06-18 22:42:47 base_handler.py:log_exception:69 WARNING] 400   POST       127.0.0.1  /index : Missing argument 'user_id'
[2023-06-18 22:42:47 main.py:log_function:25 WARNING]   6.97ms 400   POST       127.0.0.1  /index 
[2023-06-18 22:44:16 autoreload.py:_check_file:200 INFO] /Users/van/projects/likn/event-tracker/handlers/index.py modified; restarting server
[2023-06-18 22:44:17 main.py:<module>:44 INFO] servive start listening on 127.0.0.1:8090
[2023-06-18 22:44:17 index.py:post:23 INFO] request params: {'log': '', 'i': ''}
[2023-06-18 22:44:17 base_handler.py:parse_json_arguments:232 INFO] request_body:{'log': '', 'i': ''}
[2023-06-18 22:44:17 base_handler.py:parse_json_arguments:232 INFO] request_body:{'log': '', 'i': ''}
[2023-06-18 22:44:17 base_handler.py:parse_json_arguments:232 INFO] request_body:{'log': '', 'i': ''}
[2023-06-18 22:44:17 base_handler.py:log_exception:69 WARNING] 400   POST       127.0.0.1  /index : Missing argument 'user_id'
[2023-06-18 22:44:17 main.py:log_function:25 WARNING]   6.84ms 400   POST       127.0.0.1  /index 
[2023-06-18 22:44:18 index.py:post:23 INFO] request params: {'log': '', 'i': ''}
[2023-06-18 22:44:18 base_handler.py:parse_json_arguments:232 INFO] request_body:{'log': '', 'i': ''}
[2023-06-18 22:44:18 base_handler.py:parse_json_arguments:232 INFO] request_body:{'log': '', 'i': ''}
[2023-06-18 22:44:18 base_handler.py:parse_json_arguments:232 INFO] request_body:{'log': '', 'i': ''}
[2023-06-18 22:44:18 base_handler.py:log_exception:69 WARNING] 400   POST       127.0.0.1  /index : Missing argument 'user_id'
[2023-06-18 22:44:18 main.py:log_function:25 WARNING]   0.59ms 400   POST       127.0.0.1  /index 
[2023-06-18 22:45:15 main.py:<module>:44 INFO] servive start listening on 127.0.0.1:8090
[2023-06-18 22:45:18 index.py:post:23 INFO] request params: {'log': '', 'i': ''}
[2023-06-18 22:45:18 base_handler.py:parse_json_arguments:232 INFO] request_body:{'log': '', 'i': ''}
[2023-06-18 22:45:18 base_handler.py:log_exception:69 WARNING] 400   POST       127.0.0.1  /index : Missing argument 'user_id'
[2023-06-18 22:45:18 main.py:log_function:25 WARNING]   0.57ms 400   POST       127.0.0.1  /index 
[2023-06-18 22:58:25 main.py:<module>:38 INFO] servive start listening on 127.0.0.1:8090
[2023-06-18 22:58:34 index.py:post:23 INFO] request params: {'log': '', 'i': ''}
[2023-06-18 22:58:34 base_handler.py:parse_json_arguments:232 INFO] request_body:{'log': '', 'i': ''}
[2023-06-18 22:58:34 base_handler.py:log_exception:69 WARNING] 400   POST       127.0.0.1  /index : Missing argument 'user_id'
[2023-06-18 22:58:34 main.py:log_function:25 WARNING]   3.68ms 400   POST       127.0.0.1  /index 
[2023-06-18 22:58:34 index.py:post:23 INFO] request params: {'log': '', 'i': ''}
[2023-06-18 22:58:34 base_handler.py:parse_json_arguments:232 INFO] request_body:{'log': '', 'i': ''}
[2023-06-18 22:58:34 base_handler.py:log_exception:69 WARNING] 400   POST       127.0.0.1  /index : Missing argument 'user_id'
[2023-06-18 22:58:34 main.py:log_function:25 WARNING]   2.78ms 400   POST       127.0.0.1  /index 
[2023-06-18 23:00:47 main.py:<module>:38 INFO] servive start listening on 127.0.0.1:8090
[2023-06-18 23:00:54 index.py:post:23 INFO] request params: {'log': '', 'i': ''}
[2023-06-18 23:00:54 base_handler.py:parse_json_arguments:232 INFO] request_body:{'log': '', 'i': ''}
[2023-06-18 23:00:54 base_handler.py:log_exception:69 WARNING] 400   POST       127.0.0.1  /index : Missing argument 'user_id'
[2023-06-18 23:00:54 main.py:log_function:25 WARNING]   0.78ms 400   POST       127.0.0.1  /index 
[2023-06-18 23:00:55 index.py:post:23 INFO] request params: {'log': '', 'i': ''}
[2023-06-18 23:00:55 base_handler.py:parse_json_arguments:232 INFO] request_body:{'log': '', 'i': ''}
[2023-06-18 23:00:55 base_handler.py:log_exception:69 WARNING] 400   POST       127.0.0.1  /index : Missing argument 'user_id'
[2023-06-18 23:00:55 main.py:log_function:25 WARNING]   0.48ms 400   POST       127.0.0.1  /index 
[2023-06-18 23:02:02 main.py:<module>:38 INFO] servive start listening on 127.0.0.1:8090
[2023-06-18 23:02:07 main.py:<module>:38 INFO] servive start listening on 127.0.0.1:8090
[2023-06-18 23:02:10 index.py:post:23 INFO] request params: {'log': '', 'i': ''}
[2023-06-18 23:02:10 base_handler.py:parse_json_arguments:232 INFO] request_body:{'log': '', 'i': ''}
[2023-06-18 23:02:10 base_handler.py:log_exception:69 WARNING] 400   POST       127.0.0.1  /index : Missing argument 'user_id'
[2023-06-18 23:02:10 main.py:log_function:25 WARNING]   0.72ms 400   POST       127.0.0.1  /index 
[2023-06-18 23:36:46 main.py:<module>:38 INFO] servive start listening on 127.0.0.1:8090
[2023-06-18 23:36:52 index.py:post:23 INFO] request params: {'log': '', 'i': ''}
[2023-06-18 23:36:52 base_handler.py:parse_json_arguments:232 INFO] request_body:{'log': '', 'i': ''}
[2023-06-18 23:36:52 base_handler.py:log_exception:69 WARNING] 400   POST       127.0.0.1  /index : Missing argument 'user_id'
[2023-06-18 23:36:52 main.py:log_function:25 WARNING]   4.45ms 400   POST       127.0.0.1  /index 
[2023-06-18 23:42:24 main.py:<module>:38 INFO] servive start listening on 127.0.0.1:8090
[2023-06-18 23:44:41 main.py:<module>:38 INFO] servive start listening on 127.0.0.1:8090
[2023-06-18 23:52:09 main.py:<module>:38 INFO] servive start listening on 127.0.0.1:8090
2023-06-18 23:56:29-tornado.application-main.py-[line:38]-INFO-[日志信息]: servive start listening on 127.0.0.1:8090
[I 230618 23:57:22 main:38] servive start listening on 127.0.0.1:8090
2023-06-18 23:57:44-tornado.application-main.py-[line:38]-INFO-[日志信息]: servive start listening on 127.0.0.1:8090
[2023-06-18 23:57:55 main.py:<module>:38 INFO] servive start listening on 127.0.0.1:8090
[2023-06-18 23:58:47 main.py:<module>:38 INFO] servive start listening on 127.0.0.1:8090
[2023-06-18 23:59:29 main.py:<module>:38 INFO] servive start listening on 127.0.0.1:8090
[2023-06-18 23:59:50 main.py:<module>:38 INFO] 
    [0;33mservive start listening on 127.0.0.1:8090
[I 2023-06-19 00:02:21 main:38] 
    [0;33mservive start listening on 127.0.0.1:8090
[I 2023-06-19 00:02:32 main:38] 
    [0;33mservive start listening on 127.0.0.1:8090

    [0;33m [I 2023-06-19 00:04:19 main:38] 
    [0;33mservive start listening on 127.0.0.1:8090
[0;33m [I 2023-06-19 00:04:57 main:38] 
    [0;33mservive start listening on 127.0.0.1:8090
[0;33m [I 2023-06-19 00:05:26 main:38]
    [0;33mservive start listening on 127.0.0.1:8090
[0;33m[I 2023-06-19 00:05:49 main:38] 
    [0;33mservive start listening on 127.0.0.1:8090
[0;33m2023-06-19 00:07:38-tornado.application-main.py-[line:38]-INFO-[日志信息]: 
    [0;33mservive start listening on 127.0.0.1:8090
[2023-06-19 00:09:13 main.py:<module>:38 INFO] 
    [0;33mservive start listening on 127.0.0.1:8090
[2023-06-19 00:10:09 main.py:<module>:38 INFO] servive start listening on 127.0.0.1:8090
[0;33m2023-06-19 00:10:21-tornado.application-main.py-[line:38]-INFO-[日志信息]: servive start listening on 127.0.0.1:8090
[2023-06-19 00:13:12 main.py:<module>:38 INFO] servive start listening on 127.0.0.1:8090
[2023-06-19 00:16:37 main.py:<module>:38 INFO] servive start listening on 127.0.0.1:8090
[2023-06-19 00:17:12 main.py:<module>:38 INFO] servive start listening on 127.0.0.1:8090
[2023-06-19 10:27:30 main.py:<module>:38 INFO] servive start listening on 127.0.0.1:8090
[2023-06-19 10:28:59 main.py:<module>:40 INFO] servive start listening on 127.0.0.1:8090
[32m[2023-06-19 10:29:32 main.py:<module>:40 INFO](B[m servive start listening on 127.0.0.1:8090
[2023-06-19 10:30:54 main.py:<module>:40 INFO] servive start listening on 127.0.0.1:8090
[2023-06-19 10:38:18 main.py:<module>:40 INFO] servive start listening on 127.0.0.1:8090
[2023-06-19 10:38:51 main.py:<module>:40 INFO] servive start listening on 127.0.0.1:8090
[2023-06-19 10:41:41 main.py:<module>:40 INFO] servive start listening on 127.0.0.1:8090
[2023-06-19 10:47:09 index.py:post:23 INFO] request params: {'log': '', 'i': ''}
[2023-06-19 10:47:09 base_handler.py:parse_json_arguments:232 INFO] request_body:{'log': '', 'i': ''}
[2023-06-19 10:47:09 base_handler.py:log_exception:69 WARNING] 400   POST       127.0.0.1  /index : Missing argument 'user_id'
[2023-06-19 10:47:09 main.py:log_function:27 WARNING]   2.60ms 400   POST       127.0.0.1  /index 
[2023-06-19 10:48:36 main.py:<module>:40 INFO] [0;33mservive start listening on 127.0.0.1:8090
[2023-06-19 10:48:54 main.py:<module>:40 INFO] [0;33m[0;33m servive start listening on 127.0.0.1:8090
[2023-06-19 10:49:23 main.py:<module>:40 INFO] [0;33mservive start listening on 127.0.0.1:8090
[2023-06-19 10:49:56 index.py:post:23 INFO] request params: {'log': '', 'i': ''}
[2023-06-19 10:49:56 base_handler.py:parse_json_arguments:232 INFO] request_body:{'log': '', 'i': ''}
[2023-06-19 10:49:56 base_handler.py:log_exception:69 WARNING] 400   POST       127.0.0.1  /index : Missing argument 'user_id'
[2023-06-19 10:49:56 main.py:log_function:27 WARNING]   1.50ms 400   POST       127.0.0.1  /index 
[2023-06-19 10:50:47 main.py:<module>:40 INFO] 
    [0;32mservive start listening on 127.0.0.1:8090
[2023-06-19 10:50:53 index.py:post:23 INFO] request params: {'log': '', 'i': ''}
[2023-06-19 10:50:53 base_handler.py:parse_json_arguments:232 INFO] request_body:{'log': '', 'i': ''}
[2023-06-19 10:50:53 base_handler.py:log_exception:69 WARNING] 400   POST       127.0.0.1  /index : Missing argument 'user_id'
[2023-06-19 10:50:53 main.py:log_function:27 WARNING]   1.63ms 400   POST       127.0.0.1  /index 
[2023-06-19 10:51:05 main.py:<module>:40 INFO] [0;32mservive start listening on 127.0.0.1:8090
[2023-06-19 10:51:08 index.py:post:23 INFO] request params: {'log': '', 'i': ''}
[2023-06-19 10:51:08 base_handler.py:parse_json_arguments:232 INFO] request_body:{'log': '', 'i': ''}
[2023-06-19 10:51:08 base_handler.py:log_exception:69 WARNING] 400   POST       127.0.0.1  /index : Missing argument 'user_id'
[2023-06-19 10:51:08 main.py:log_function:27 WARNING]   1.64ms 400   POST       127.0.0.1  /index 
[2023-06-19 10:52:14 index.py:post:23 INFO] request params: {'log': '', 'i': ''}
[2023-06-19 10:52:14 base_handler.py:parse_json_arguments:232 INFO] request_body:{'log': '', 'i': ''}
[2023-06-19 10:52:14 base_handler.py:log_exception:69 WARNING] 400   POST       127.0.0.1  /index : Missing argument 'user_id'
[2023-06-19 10:52:14 main.py:log_function:27 WARNING]   0.49ms 400   POST       127.0.0.1  /index 
[2023-06-19 10:52:15 index.py:post:23 INFO] request params: {'log': '', 'i': ''}
[2023-06-19 10:52:15 base_handler.py:parse_json_arguments:232 INFO] request_body:{'log': '', 'i': ''}
[2023-06-19 10:52:15 base_handler.py:log_exception:69 WARNING] 400   POST       127.0.0.1  /index : Missing argument 'user_id'
[2023-06-19 10:52:15 main.py:log_function:27 WARNING]   0.49ms 400   POST       127.0.0.1  /index 
[2023-06-19 10:53:18 main.py:<module>:40 INFO] 
    [0;32mservive start listening on 127.0.0.1:8090
[2023-06-19 10:53:47 main.py:<module>:40 INFO] [0;32mservive start listening on 127.0.0.1:8090
[2023-06-19 10:53:55 main.py:<module>:40 INFO] \033[0;32mservive start listening on 127.0.0.1:8090
[2023-06-19 10:54:01 main.py:<module>:40 INFO] [0;32mservive start listening on 127.0.0.1:8090
 2023-06-19 11:08:50 I   [0;32mservive start listening on 127.0.0.1:8090
 2023-06-19 11:12:47 I   [0;32mservive start listening on 127.0.0.1:8090
 2023-06-19 16:35:03 I   [0;32mservive start listening on 127.0.0.1:8090
 2023-06-19 16:36:58 I   [0;32mservive start listening on 127.0.0.1:8090
 2023-06-19 16:37:43 I   [0;32mservive start listening on 127.0.0.1:8090
 2023-06-19 17:30:42 I   [0;32mservive start listening on 127.0.0.1:8090
 2023-06-20 11:56:14 I   [0;32mservive start listening on 127.0.0.1:8090
 2023-06-20 11:57:31 I   [0;32mservive start listening on 127.0.0.1:8090
 2023-06-20 12:01:12 I   [0;32mservive start listening on 127.0.0.1:8090
 2023-06-20 14:50:10 I   [0;32mservive start listening on 127.0.0.1:8090
 2023-06-20 14:51:14 I   [0;32mservive start listening on 127.0.0.1:8090
 2023-06-20 14:51:57 I   [0;32mservive start listening on 127.0.0.1:8090
 2023-06-20 15:28:15 I   [0;32mservive start listening on 127.0.0.1:8090
 2023-06-20 15:53:38 I   [0;32mservive start listening on 127.0.0.1:8090
 2023-06-20 15:54:25 I   [0;32mservive start listening on 127.0.0.1:8090
 2023-06-20 15:58:36 I   [0;32mservive start listening on 127.0.0.1:8090
 2023-06-20 16:23:07 I   [0;32mservive start listening on 127.0.0.1:8090
 2023-06-21 10:31:51 I   [0;32mservive start listening on 127.0.0.1:8090
 2023-06-21 10:34:54 I   [0;32mservive start listening on 127.0.0.1:8090
 2023-06-21 10:37:36 W     0.92ms 405    GET       127.0.0.1  /nft/event_logs 
 2023-06-21 10:37:50 I   request params: {'contract_address': '0xB66a603f4cFe17e3D27B87a8BfCaD319856518B8', 'token_id': '93120456044919683586133790223573698740379388494099189275018955947239192657923'}
 2023-06-21 10:37:50 I   request_body:{'contract_address': '0xB66a603f4cFe17e3D27B87a8BfCaD319856518B8', 'token_id': '93120456044919683586133790223573698740379388494099189275018955947239192657923'}
 2023-06-21 10:37:50 I    41.40ms 200   POST       127.0.0.1  /nft/event_logs 
 2023-06-21 10:37:52 I   request params: {'contract_address': '0xB66a603f4cFe17e3D27B87a8BfCaD319856518B8', 'token_id': '93120456044919683586133790223573698740379388494099189275018955947239192657923'}
 2023-06-21 10:37:52 I   request_body:{'contract_address': '0xB66a603f4cFe17e3D27B87a8BfCaD319856518B8', 'token_id': '93120456044919683586133790223573698740379388494099189275018955947239192657923'}
 2023-06-21 10:37:52 I     6.09ms 200   POST       127.0.0.1  /nft/event_logs 
 2023-06-21 10:42:06 I   [0;32mservive start listening on 127.0.0.1:8090
 2023-06-21 10:42:10 I   request params: {'contract_address': '0xB66a603f4cFe17e3D27B87a8BfCaD319856518B8', 'token_id': '93120456044919683586133790223573698740379388494099189275018955947239192657923'}
 2023-06-21 10:42:10 I   request_body:{'contract_address': '0xB66a603f4cFe17e3D27B87a8BfCaD319856518B8', 'token_id': '93120456044919683586133790223573698740379388494099189275018955947239192657923'}
 2023-06-21 10:42:10 E   Uncaught exception   POST       127.0.0.1  /nft/event_logs 
    HTTPServerRequest(protocol='http', host='127.0.0.1:8090', method='POST', uri='/nft/event_logs', version='HTTP/1.1', remote_ip='127.0.0.1')
    Traceback (most recent call last):
      File "/Users/van/projects/likn/event-tracker/venv/lib/python3.9/site-packages/tornado/web.py", line 1786, in _execute
        result = await result
      File "/Users/van/projects/likn/event-tracker/venv/lib/python3.9/site-packages/tornado/gen.py", line 786, in run
        yielded = self.gen.send(value)
      File "/Users/van/projects/likn/event-tracker/handlers/event_log_handler.py", line 20, in post
        self.success(data=res)
      File "/Users/van/projects/likn/event-tracker/handlers/base_handler.py", line 140, in success
        self.finish_with_json(dict(status=0, msg=msg, data=data))
      File "/Users/van/projects/likn/event-tracker/handlers/base_handler.py", line 301, in finish_with_json
        raise Finish(json.dumps(data).encode())
      File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/json/__init__.py", line 231, in dumps
        return _default_encoder.encode(obj)
      File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/json/encoder.py", line 199, in encode
        chunks = self.iterencode(o, _one_shot=True)
      File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/json/encoder.py", line 257, in iterencode
        return _iterencode(o, 0)
      File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/json/encoder.py", line 179, in default
        raise TypeError(f'Object of type {o.__class__.__name__} '
    TypeError: Object of type ObjectId is not JSON serializable
 2023-06-21 10:42:10 E    41.19ms 500   POST       127.0.0.1  /nft/event_logs 
 2023-06-21 10:45:22 I   [0;32mservive start listening on 127.0.0.1:8090
 2023-06-21 10:45:38 I   [0;32mservive start listening on 127.0.0.1:8090
 2023-06-21 10:45:41 I   request params: {'contract_address': '0xB66a603f4cFe17e3D27B87a8BfCaD319856518B8', 'token_id': '93120456044919683586133790223573698740379388494099189275018955947239192657923'}
 2023-06-21 10:45:41 I   request_body:{'contract_address': '0xB66a603f4cFe17e3D27B87a8BfCaD319856518B8', 'token_id': '93120456044919683586133790223573698740379388494099189275018955947239192657923'}
 2023-06-21 10:45:41 E   Uncaught exception   POST       127.0.0.1  /nft/event_logs 
    HTTPServerRequest(protocol='http', host='127.0.0.1:8090', method='POST', uri='/nft/event_logs', version='HTTP/1.1', remote_ip='127.0.0.1')
    Traceback (most recent call last):
      File "/Users/van/projects/likn/event-tracker/venv/lib/python3.9/site-packages/tornado/web.py", line 1786, in _execute
        result = await result
      File "/Users/van/projects/likn/event-tracker/venv/lib/python3.9/site-packages/tornado/gen.py", line 786, in run
        yielded = self.gen.send(value)
      File "/Users/van/projects/likn/event-tracker/handlers/event_log_handler.py", line 18, in post
        self.success(data=res)
      File "/Users/van/projects/likn/event-tracker/handlers/base_handler.py", line 140, in success
        self.finish_with_json(dict(status=0, msg=msg, data=data))
      File "/Users/van/projects/likn/event-tracker/handlers/base_handler.py", line 301, in finish_with_json
        raise Finish(json.dumps(data).encode())
      File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/json/__init__.py", line 231, in dumps
        return _default_encoder.encode(obj)
      File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/json/encoder.py", line 199, in encode
        chunks = self.iterencode(o, _one_shot=True)
      File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/json/encoder.py", line 257, in iterencode
        return _iterencode(o, 0)
      File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/json/encoder.py", line 179, in default
        raise TypeError(f'Object of type {o.__class__.__name__} '
    TypeError: Object of type ObjectId is not JSON serializable
 2023-06-21 10:45:41 E    32.99ms 500   POST       127.0.0.1  /nft/event_logs 
 2023-06-21 10:50:55 I   [0;32mservive start listening on 127.0.0.1:8090
 2023-06-21 10:51:02 I   request params: {'contract_address': '0xB66a603f4cFe17e3D27B87a8BfCaD319856518B8', 'token_id': '93120456044919683586133790223573698740379388494099189275018955947239192657923'}
 2023-06-21 10:51:02 I   request_body:{'contract_address': '0xB66a603f4cFe17e3D27B87a8BfCaD319856518B8', 'token_id': '93120456044919683586133790223573698740379388494099189275018955947239192657923'}
 2023-06-21 10:51:02 E   Uncaught exception   POST       127.0.0.1  /nft/event_logs 
    HTTPServerRequest(protocol='http', host='127.0.0.1:8090', method='POST', uri='/nft/event_logs', version='HTTP/1.1', remote_ip='127.0.0.1')
    Traceback (most recent call last):
      File "/Users/van/projects/likn/event-tracker/venv/lib/python3.9/site-packages/tornado/web.py", line 1786, in _execute
        result = await result
      File "/Users/van/projects/likn/event-tracker/venv/lib/python3.9/site-packages/tornado/gen.py", line 786, in run
        yielded = self.gen.send(value)
      File "/Users/van/projects/likn/event-tracker/handlers/event_log_handler.py", line 18, in post
        self.success(data=res)
      File "/Users/van/projects/likn/event-tracker/handlers/base_handler.py", line 140, in success
        self.finish_with_json(dict(status=0, msg=msg, data=data))
      File "/Users/van/projects/likn/event-tracker/handlers/base_handler.py", line 301, in finish_with_json
        raise Finish(json.dumps(data).encode())
      File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/json/__init__.py", line 231, in dumps
        return _default_encoder.encode(obj)
      File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/json/encoder.py", line 199, in encode
        chunks = self.iterencode(o, _one_shot=True)
      File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/json/encoder.py", line 257, in iterencode
        return _iterencode(o, 0)
      File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/json/encoder.py", line 179, in default
        raise TypeError(f'Object of type {o.__class__.__name__} '
    TypeError: Object of type datetime is not JSON serializable
 2023-06-21 10:51:02 E    54.35ms 500   POST       127.0.0.1  /nft/event_logs 
 2023-06-21 10:51:36 I   [0;32mservive start listening on 127.0.0.1:8090
 2023-06-21 10:51:41 I   request params: {'contract_address': '0xB66a603f4cFe17e3D27B87a8BfCaD319856518B8', 'token_id': '93120456044919683586133790223573698740379388494099189275018955947239192657923'}
 2023-06-21 10:51:41 I   request_body:{'contract_address': '0xB66a603f4cFe17e3D27B87a8BfCaD319856518B8', 'token_id': '93120456044919683586133790223573698740379388494099189275018955947239192657923'}
 2023-06-21 10:51:41 I    36.96ms 200   POST       127.0.0.1  /nft/event_logs 
 2023-06-21 10:56:11 E   Uncaught exception   POST       127.0.0.1  /nft/event_logs 
    HTTPServerRequest(protocol='http', host='127.0.0.1:8090', method='POST', uri='/nft/event_logs', version='HTTP/1.1', remote_ip='127.0.0.1')
    Traceback (most recent call last):
      File "/Users/van/projects/likn/event-tracker/venv/lib/python3.9/site-packages/tornado/web.py", line 1786, in _execute
        result = await result
      File "/Users/van/projects/likn/event-tracker/venv/lib/python3.9/site-packages/tornado/gen.py", line 234, in wrapper
        yielded = ctx_run(next, result)
      File "/Users/van/projects/likn/event-tracker/handlers/event_log_handler.py", line 13, in post
        app_log.info(f"request params: {json.loads(self.request.body.decode())}")
      File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/json/__init__.py", line 346, in loads
        return _default_decoder.decode(s)
      File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/json/decoder.py", line 337, in decode
        obj, end = self.raw_decode(s, idx=_w(s, 0).end())
      File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/json/decoder.py", line 353, in raw_decode
        obj, end = self.scan_once(s, idx)
    json.decoder.JSONDecodeError: Expecting property name enclosed in double quotes: line 4 column 1 (char 76)
 2023-06-21 10:56:11 E    12.52ms 500   POST       127.0.0.1  /nft/event_logs 
 2023-06-21 10:56:51 I   request params: {'contract_address': '0xB66a603f4cFe17e3D27B87a8BfCaD319856518B8'}
 2023-06-21 10:56:51 I   request_body:{'contract_address': '0xB66a603f4cFe17e3D27B87a8BfCaD319856518B8'}
 2023-06-21 10:56:51 W   400   POST       127.0.0.1  /nft/event_logs : Missing argument 'token_id'
 2023-06-21 10:56:51 W     1.61ms 400   POST       127.0.0.1  /nft/event_logs 
 2023-06-21 10:56:54 I   request params: {'contract_address': '0xB66a603f4cFe17e3D27B87a8BfCaD319856518B8'}
 2023-06-21 10:56:54 I   request_body:{'contract_address': '0xB66a603f4cFe17e3D27B87a8BfCaD319856518B8'}
 2023-06-21 10:56:54 W   400   POST       127.0.0.1  /nft/event_logs : Missing argument 'token_id'
 2023-06-21 10:56:54 W     1.72ms 400   POST       127.0.0.1  /nft/event_logs 
 2023-06-21 10:57:13 I   request params: {'contract_address': '0xB66a603f4cFe17e3D27B87a8BfCaD319856518B8'}
 2023-06-21 10:57:13 I   request_body:{'contract_address': '0xB66a603f4cFe17e3D27B87a8BfCaD319856518B8'}
 2023-06-21 10:57:13 W   400   POST       127.0.0.1  /nft/event_logs : Missing argument 'token_id'
 2023-06-21 10:57:13 W     1.20ms 400   POST       127.0.0.1  /nft/event_logs 
 2023-06-21 11:01:37 I   request params: {'contract_address': '0xB66a603f4cFe17e3D27B87a8BfCaD319856518B8', 'token_id': '93120456044919683586133790223573698740379388494099189275018955947239192657923'}
 2023-06-21 11:01:37 I   request_body:{'contract_address': '0xB66a603f4cFe17e3D27B87a8BfCaD319856518B8', 'token_id': '93120456044919683586133790223573698740379388494099189275018955947239192657923'}
 2023-06-21 11:01:37 I     6.00ms 200   POST       127.0.0.1  /nft/event_logs 
 2023-06-21 11:06:08 I   [0;32mservive start listening on 127.0.0.1:8090
 2023-06-21 11:06:15 I   request params: {'contract_address': '0xB66a603f4cFe17e3D27B87a8BfCaD319856518B8'}
 2023-06-21 11:06:15 I   request_body:{'contract_address': '0xB66a603f4cFe17e3D27B87a8BfCaD319856518B8'}
 2023-06-21 11:08:00 W   400   POST       127.0.0.1  /nft/event_logs : Missing argument 'token_id'
 2023-06-21 11:08:09 W   114549.76ms 400   POST       127.0.0.1  /nft/event_logs 
 2023-06-21 11:08:39 I   [0;32mservive start listening on 127.0.0.1:8090
 2023-06-21 11:08:44 I   [0;32mservive start listening on 127.0.0.1:8090
 2023-06-21 11:08:48 I   request params: {'contract_address': '0xB66a603f4cFe17e3D27B87a8BfCaD319856518B8'}
 2023-06-21 11:08:48 I   request_body:{'contract_address': '0xB66a603f4cFe17e3D27B87a8BfCaD319856518B8'}
 2023-06-21 11:08:48 W   400   POST       127.0.0.1  /nft/event_logs : Missing argument 'token_id'
 2023-06-21 11:08:48 W     1.92ms 400   POST       127.0.0.1  /nft/event_logs 
 2023-06-21 11:08:51 I   request params: {'contract_address': '0xB66a603f4cFe17e3D27B87a8BfCaD319856518B8'}
 2023-06-21 11:08:51 I   request_body:{'contract_address': '0xB66a603f4cFe17e3D27B87a8BfCaD319856518B8'}
 2023-06-21 11:08:51 W   400   POST       127.0.0.1  /nft/event_logs : Missing argument 'token_id'
 2023-06-21 11:08:51 W     4.93ms 400   POST       127.0.0.1  /nft/event_logs 
 2023-06-21 11:16:28 I   [0;32mservive start listening on 127.0.0.1:8090
 2023-06-21 11:31:39 I   request params: {'contract_address': '0xA2D9Ded6115b7B7208459450D676f0127418ae7A', 'token_id': '37026241720571211328131996875863618460101766320746399388549332128303216066562'}
 2023-06-21 11:31:39 I   request_body:{'contract_address': '0xA2D9Ded6115b7B7208459450D676f0127418ae7A', 'token_id': '37026241720571211328131996875863618460101766320746399388549332128303216066562'}
 2023-06-21 11:31:39 I    30.16ms 200   POST       127.0.0.1  /nft/event_logs 
 2023-06-21 11:32:03 I   request params: {'contract_address': '0xA2D9Ded6115b7B7208459450D676f0127418ae7A', 'token_id': '51032156517012602710750852899093911391733291613482576614848296747610922811393'}
 2023-06-21 11:32:03 I   request_body:{'contract_address': '0xA2D9Ded6115b7B7208459450D676f0127418ae7A', 'token_id': '51032156517012602710750852899093911391733291613482576614848296747610922811393'}
 2023-06-21 11:32:03 I    24.64ms 200   POST       127.0.0.1  /nft/event_logs 
